app/components/support_interface/audit_trail_component.rb

```diff
@@ -9,9 +9,21 @@ def initialize(audited_thing:)
     end
 
     def audits
-      audited_thing.own_and_associated_audits.includes(:user).order('id desc')
+      return audited_thing.own_and_associated_audits.includes(:user).order('id desc') unless audited_thing.is_a? Provider
+
+      (audited_thing.own_and_associated_audits.includes(:user) + relationship_permissions_audits_for_ratifying_provider)
+      .sort_by { |audit| -audit[:id] }
     end
 
     attr_reader :audited_thing
+
+  private
+
+    def relationship_permissions_audits_for_ratifying_provider
+      Audited::Audit.where(
+        auditable_id: ProviderRelationshipPermissions.where(ratifying_provider_id: audited_thing.id),
```

Comment:
Should we also restrict this by `auditable_type: "ProviderRelationshipPermissions"`?